composer/utils/s3_object_store.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Is it possible to add test cases for this? Take a look at https://github.com/spulec/moto for mocking out s3